src/appleseed.studio/help/about/aboutwindow.cpp

```diff
@@ -127,11 +127,27 @@ void AboutWindow::set_library_features()
         false;
 #endif
 
+    const bool WithSpectralSupport =
+#ifdef APPLESEED_WITH_SPECTRAL_SUPPORT
+        true;
+#else
+        false;
+#endif
+
+    const bool WithGPUSupport =
+#ifdef APPLESEED_WITH_GPU
+        true;
+#else
+        false;
+#endif
+
     QString details;
     details += "This build of appleseed has the following features:\n\n";
     details += QString("  Instruction sets: %1\n").arg(Appleseed::get_lib_cpu_features());
-    details += QString("  Disney material: %1\n").arg(to_enabled_disabled(WithDisneyMaterial));
+    details += QString("  Disney SeExpr: %1\n").arg(to_enabled_disabled(WithDisneyMaterial));
```

Comment:
"Disney SeExpr" isn't very clear either. Maybe "Disney built-in material"?

Reply:
It should not lead to confusion with our Disney shaders. So I would prefer the name to not contain `material`. 
As I understood it, enabling this is only needed when one wants to use SeExpr. It does not affect the building of the asDisneyMaterial OSL shader or legacy appleseedDisney one.

Reply:
Then maybe "Disney material with SeExpr support"? My problem right now is that "Disney" alone doesn't mean much, and that "Disney SeExpr" is not very clear.

Reply:
This is indeed better. I changed it.

Reply:
Added the changes also for the cli version info.